weave-js/src/components/PagePanelComponents/Home/Browse3/typeViews/PIL.Image.Image/PILImageImage.tsx

```diff
@@ -21,11 +21,25 @@ export const PILImageImage: React.FC<{
   data: PILImageImageTypePayload;
 }> = props => {
   const {useFileContent} = useWFHooks();
+
+  const imageTypes = {
+    'image.jpg': 'jpg',
+    'image.png': 'png',
+  } as const;
+
+  const imageKey = Object.keys(props.data.files).find(
+    key => key in imageTypes
+  ) as keyof PILImageImageTypePayload['files'];
+  if (!imageKey) {
+    throw new Error('No image file found');
```

Comment:
def don't throw. just render a empty / some info state. Throwing will crash the page